.projen/deps.json

```diff
@@ -97,7 +97,7 @@
     },
     {
       "name": "aws-cdk-lib",
-      "version": "^2.1.0",
+      "version": "^2.34.0",
```

Comment:
Should we accelerate to `2.38.0` while we are at it? Appears to be [`cdk@latest`](https://docs.aws.amazon.com/cdk/api/versions.html)